tests/installation/ipxe_install.pm

```diff
@@ -70,13 +70,23 @@ sub set_bootscript {
     my $arch = get_required_var('ARCH');
     my $autoyast = get_var('AUTOYAST', '');
     my $mirror_http = get_required_var('MIRROR_HTTP');
+    my $openqa_hostname = get_var('OPENQA_HOSTNAME', 'openqa.suse.de');
 
     # trim all strings from variables to get rid of bogus whitespaces
     $arch =~ s/^\s+|\s+$//g;
     $autoyast =~ s/^\s+|\s+$//g;
     $mirror_http =~ s/^\s+|\s+$//g;
+    $openqa_hostname =~ s/^\s+|\s+$//g;
 
+    # For pxe/ipxe agama installation,we don't have linuxrc 'install=' parameter
+    # so we need to pass iso image to boot
     my $install = "install=$mirror_http";
+    if (is_agama) {
+        my $agama_iso = get_required_var('ISO');
+        $agama_iso =~ s/^\s+|\s+$//g;
+        my $agama_live_iso = "http://$openqa_hostname/assets/iso/$agama_iso";
```

Comment:
I personally prefer more flexible installation source instead of just openQA, especially when we do verification on personal openQA.

Reply:
Thanks @waynechen55, let me add a switch here :)